haproxy/assets/configuration/spec.yaml

```diff
@@ -4,10 +4,17 @@ files:
   options:
   - template: init_config
     options:
-    - template: init_config/http
-    - template: init_config/default
+    - template: init_config/openmetrics
   - template: instances
     options:
+    - name: use_preview
```

Comment:
Can we be more explicit and name it `use_prometheus`? `preview` is going to get outdated real quick

Reply:
This is what we usually do: https://github.com/DataDog/integrations-core/blob/eaa2a61e44a8d4f1ccea723238a81cb3647bfd14/etcd/datadog_checks/etcd/data/conf.yaml.example#L26-L29

Prometheus is just an implementation detail imo. WDYT?

Reply:
I know its what we usually do, I don't think is good naming in any case cause it does not age well. It is also not just _an implementation detail_ because the user needs to provide either an API url or a prometheus_url so it's not something that is purely implementation choice and the user is not aware of, quite the oposite.
Also _preview_ implies that is a beta of a new version or something on those lines and prometheus integrations are anything but betas

Reply:
Actually, if we think in terms of the feature rather than the implementation, it very much is beta. As I said in OP:

> dropping a few options (one of which was already deprecated) and will iteratively add back any based on feedback from users